core/scripts/bareos-config-lib.sh.in

```diff
@@ -234,6 +234,78 @@ get_database_utility_path()
     fi
 }
 
+_scsicrypto_cmds="bareos-sd bcopy bextract bls bscan bscrypto btape"
+_scsicrypto_config_file="${BAREOS_CONFIG_DIR}/.enable-capabilities"
+check_scsicrypto_capabilities()
+{
+  if [ "${os_type}" != "Linux" ];then
+      error "check_scsicrypto_capabilities() is not supported on this platform"
+      return 1
+  fi
+  if ( [ "$(command -v getcap)" = "" ] || [ "$(command -v setcap)" = "" ] ); then
+      error "getcap and setcap commands are mandatory"
+      return 1
+  fi
+  if [ -f "${_scsicrypto_config_file}" ];then
+    cap_missing_conf=0
+  else
+    cap_missing_conf=1
+  fi
+  for c in ${_scsicrypto_cmds};do
+    if ! r="$(setcap -v cap_sys_rawio=ep "${BAREOS_SBIN_DIR}/${c}")"; then
+       warn "${r}"
+       cap_missing_conf=1
+    fi
+  done
+  if [ ${cap_missing_conf} -ne 0 ];then
+    warn "One of the tools are not configured for scsi crypto."
+    warn "Run set_scsicrypto_capabilities to fix capabilities."
+  else
+    info "All tools have cap_sys_rawio=ep set."
+  fi
+}
+
+set_scsicrypto_capabilities()
+{
+  if [ "${os_type}" != "Linux" ];then
+      error "set_scsicrypto_capabilities() is not supported on this platform"
+      return 1
+  fi
+  for c in ${_scsicrypto_cmds};do
+    if ! r="$(setcap cap_sys_rawio=ep "${BAREOS_SBIN_DIR}/${c}")"; then
+       error "setcap on ${c} has failed - ${r}"
+       return 1
+    else
+      if ! r="$(chmod 0750 "${BAREOS_SBIN_DIR}/${c}")"; then
+         error "Adjusting chmod failed on ${c}"
+         return 1
+      fi
```

Comment:
If you have two separate checks here, you might want to change the order (first chmod 0750, than add capabities).
On Debian, the binaries do belong to root, not the user Bareos. I propose:

```
chgrp bareos $i
chmod o-rwx $i
setcap cap_sys_rawio=ep $i
```

Reply:
Ok make sense. I wasn't aware about the difference in Debian.